src/components/UpcomingEvents/index.jsx

```diff
@@ -95,8 +95,8 @@ const Events = ({ data, slideStart, loading }) => {
       className="col-start-1 articles-component col-end-8 flex flex-col justify-between lg:pr-8"
       style={slideStart}
     >
-      {data.map(eventData => {
-        return <Event data={eventData} loading={loading} />;
+      {data.map((eventData, index) => {
+        return <Event key={index} data={eventData} loading={loading} />;
```

Comment:
It is a good practice to use ID instead of `index`

Reply:
there is no id here .. the data passed as a prop to the child

Reply:
You can close pull request I discovered I have fixed the issue in the widgets branch

Reply:
> there is no id here .. the data passed as a prop to the child

If I think `eventData` is an object that contains event details. so it should have ID.
🤔